bevy_lint/action.yml

```diff
@@ -9,6 +9,28 @@ description: |
 
   Note that this currently will override the default Rustup toolchain to nightly. You can fix this
   by calling `rustup default` after this action is run.
+inputs:
+  cache:
+    description: |
+      If set to `true`, this action will cache the built `bevy_lint` executable and reuse it on
+      subsequent runs, which can speed up the runtime. This defaults to `false`.
+    required: false
+    default: 'false'
```

Comment:
I'm curious, if the speedup is so big, why don't we enable caching by default?
I would expect most users wanting to have it enabled.